product_multi_price/models/product_pricelist.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = 'product.pricelist.item'
+
+    def _get_multi_price_field_names(self):
```

Comment:
You just need to inherit `_get_multi_price_field_names`, call super, and add what you want.